From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Mon, 22 Aug 2022 08:31:15 -0400
Subject: firmware-security: Disable

Not yet useful enough to justify its inclusion by default here

You can get the same info by running
fwupdmgr security

If you drop this patch, add a Depends: fwupd [linux-any]

https://launchpad.net/bugs/1987162

https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1938
---
 shell/cc-panel-loader.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell/cc-panel-loader.c b/shell/cc-panel-loader.c
index 17f9601..1b819a5 100644
--- a/shell/cc-panel-loader.c
+++ b/shell/cc-panel-loader.c
@@ -113,7 +113,7 @@ static CcPanelLoaderVtable default_panels[] =
   PANEL_TYPE("info-overview",    cc_info_overview_panel_get_type,        NULL),
   PANEL_TYPE("keyboard",         cc_keyboard_panel_get_type,             NULL),
   PANEL_TYPE("location",         cc_location_panel_get_type,             NULL),
-  PANEL_TYPE("firmware-security",cc_firmware_security_panel_get_type,    cc_firmware_security_panel_static_init_func),
+  // PANEL_TYPE("firmware-security",cc_firmware_security_panel_get_type,    cc_firmware_security_panel_static_init_func),
   PANEL_TYPE("microphone",       cc_microphone_panel_get_type,           NULL),
   PANEL_TYPE("mouse",            cc_mouse_panel_get_type,                NULL),
   PANEL_TYPE("multitasking",     cc_multitasking_panel_get_type,         NULL),
